update-pot

```diff
@@ -3,6 +3,28 @@
 
 set -e
 
+# In LP#1758684 we got reports that the pot file generation
+# is broken. To get to the bottom of this add checks here
+# so that we error the build if this happens. Note that the
+# strings may be update if those change but spread tests will
+# tell us when it is needed.
+check_canaries() {
+    c1="Alternative command to run"
+    c2="Name of the key to use, otherwise use the default key"
+    c3="too many arguments for command"
+
+    for canary in "$c1" "$c2" "$c3"; do
+	if ! grep -q "$canary" "$OUTPUT"; then
+	    echo "canary '$canary' not found, pot extraction broken"
+	    ls -lh "$OUTPUT"
+	    cat "$OUTPUT"
+	    echo "Find found:"
+	    find "$HERE" -name "*.go" -type f
```

Comment:
Shouldn't this be `find "$HERE" -type d \( -name "vendor" -o -name "_build" \) -prune -o -name "*.go" -type f` like it's below?

Reply:
Yes, I removed this again now - now that the reason for the failure is understood I think this is overkill.